cosima_cookbook/database.py

```diff
@@ -61,6 +61,7 @@ class NCExperiment(Base):
     #: Short, categorical keywords
     kw = relationship(
         'Keyword',
+        lazy='joined',
```

Comment:
Sure. If you do that I'll rebase on your change.

Reply:
Actually, I don't think this line is needed at all now, since the filtering is on `NCExperiment.keywords`, rather than `Keyword`, so it'll trigger the default lazy loading through a subquery.

Reply:
You're right. I removed it and it is all sweet.